examples/sample-app/application-template-dockerbuild.json

```diff
@@ -83,14 +83,14 @@
       "kind": "ImageStream",
       "apiVersion": "v1",
       "metadata": {
-        "name": "ruby-25-centos7"
+        "name": "ruby-27-centos7"
       },
       "spec": {
         "tags": [
           {
             "from": {
               "kind": "DockerImage",
-              "name": "docker.io/centos/ruby-25-centos7:latest"
+              "name": "docker.io/centos/ruby-27-centos7:latest"
```

Comment:
Due to docker hub pull limit, we may use quay.io image `quay.io/centos7/ruby-27-centos7`